main.cpp

```diff
@@ -52,84 +53,110 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: bicycle
+//  action 1: the bicycle rides
+bicycle.ride();
+//  action 2: the bicycle flies
+bicycle.fly();
+//  action 3: the bicycle dings its bell
+bicycle.bells();

```

Comment:
the action is `dings`
the pseudo code needs to say this.
`bells` is not an action, it is a noun.
- [ ] revise